miio/vacuum_cli.py

```diff
@@ -557,6 +561,25 @@ def carpet_mode(vac: miio.Vacuum, enabled=None):
         click.echo(vac.set_carpet_mode(enabled))
 
 
+@cli.command()
+@click.argument("mode", required=False, type=str)
+@pass_dev
+def carpet_cleaning_mode(vac: miio.Vacuum, mode=None):
+    """Query or set the carpet cleaning/avoidance mode.
+
+    Allowed values: Avoid, Rise, Ignore
+    """
+    from miio.vacuum import CarpetCleaningMode
```

Comment:
Move the import to the top of the file among other imports.

Reply:
I explicitly didn't because that's the way it is handled in `reset_consumable()`. I don't mind moving the import to the top though